core/src/main/kotlin/version/Handler.kt

```diff
@@ -0,0 +1,23 @@
+package com.epam.drill.admin.version
+
+import com.epam.drill.admin.endpoints.*
+import com.epam.drill.admin.plugins.*
+
+object AdminVersion {
```

Comment:
What is the reason for making another object with exact same fields as in AdminVersionDto?